CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 #### Visual Environment
 
+- [Magnet Alignment algorithm is used while placing new nodes][3366]. The new
+  node position now depends on the positions of the surrounding nodes. This
+  makes the resulting graph prettier.
```

Comment:
This description does not really tell what is happening here. I'd add some info about empty space on the screen and why we do it.

Reply:
I'm not sure how to express it shortly, so I ask for help if you don't like an updated description.